app/javascript/react/screens/App/Plan/components/PlanRequestDetailList/PlanRequestDetailList.js

```diff
@@ -520,6 +657,23 @@ class PlanRequestDetailList extends React.Component {
             onSubmit={this.onSubmit}
           />
         </div>
+        <ConfirmModal
+          show={showConfirmCancel}
+          title={__('Cancel Migrations')}
+          body={
+            <React.Fragment>
+              <p>
+                {__('If you cancel, these VMs will not be migrated and will be fully restored in the source provider.')}
+              </p>
+              <ul>{selectedCancelTasks.map(task => <li key={task.id}>{task.vmName}</li>)}</ul>
+              <p>{__('Do you want to cancel?')}</p>
+            </React.Fragment>
+          }
+          cancelButtonLabel={__('No')}
+          confirmButtonLabel={__('Cancel Migrations')}
```

Comment:
@AllenBW This is a classic UX conundrum - how to confirm an action when that action is cancelling something. The normal button labels (OK, Cancel) lead to confusion - Does Cancel cancel or cancel the Cancel. So I ran this by Stacey (our info architect) and this is what we came up with. Maybe not consistent, but in this case necessary.

Reply:
@AllenBW @vconzola Just my $0.02, but I feel like the single-word restriction is specifically confusing in this "cancelling the cancel" case. I've been happy with products that I see use the opposite tactic when one word isn't enough-- very explicit and verbose buttons.

"Yes, cancel these migrations"   /   "No, do not cancel anything"   or something to that effect.

I think using one word where it's not confusing, and then just going straight for a sentence otherwise instead of trying to find the two word button that works, is good.  End of non-designer opinion.